### PR TITLE
Add new file for new CJ

### PR DESCRIPTION
**Pull request by ChrisJergens**
Please accept my cool file for you branch

### CODE REVIEW

**Comment by ChrisJergens** [comment]
#29

**Comment by ChrisJergens** [comment]
resovle #29

File: ChrisJfile.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+## a New file for my bio
```
Comment (kpineiro) [inline ChrisJfile.txt:1]: @ChrisJergens  Looks like its missing something ;) ?

**Comment by jaygarala** [comment]
typo 'resolve'

File: ChrisJfile.txt (hunk 1 above)
Comment (rsaVolk) [inline ChrisJfile.txt:1]: @ChrisJergens, Looks about as detailed as mine, not too much?

**Comment by ChrisJergens** [comment]
resolve

**Comment by ChrisJergens** [comment]
@kevindouglas2 are you going to accept my PR or not?  Do you not like my shoes or somethings?

**Pull request merged**